app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_name = command.split()
+    if file_name[1] != file_name[2]:
```

Comment:
Make sure you check that the `cp` input command is correct.

Reply:
in this condition I check if first file name = second file name, code stops